client/ruby/hostedtest_api.rb

```diff
@@ -330,7 +306,10 @@ def create_upstream_subscription(subscription_id, owner_key, params = {})
       raise "Attempting to create a subscription without a product"
     end
 
-    return @cp.post('hostedtest/subscriptions', {}, subscription)
+    # TODO: We're defaulting to creating children on creation to maintain existing behavior
+    # form ancient tests. When the hosted tests are ported to Java, this behavior should not
```

Comment:
Typo: "form" should be "from" and the same needs to be changed on line 341

Reply:
fixed